.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -55,6 +55,7 @@ body:
         - supertinycron
         - thorvg
         - zlib
+        - libjpeg-turbo
```

Comment:
Since library names are automatically prefixed with `lib` when building, I would suggest naming the component `jpeg-turbo`. The resulting library will be `libjpeg-turbo.a`.

Reply:
Ah, the upstream repo is called the same way... Okay, let's follow the upstream naming then.